docs/en/deploy/Running-Alluxio-On-Docker.md

```diff
@@ -7,7 +7,7 @@ priority: 4
 ---
 
 Docker can be used to simplify the deployment and management of Alluxio servers.
-Using the [alluxio/alluxio](https://hub.docker.com/r/alluxio/alluxio/) Docker
+Using the [alluxio/{{site.ALLUXIO_DOCKER_IMAGE}}](https://hub.docker.com/r/alluxio/{{site.ALLUXIO_DOCKER_IMAGE}}/) Docker
```

Comment:
why we want to replace this with a new macro? do we plan to use a different docker image name in the future?